service/local-dev/run_server.sh

```diff
@@ -1,34 +1,45 @@
-#!/bin/bash
+#!/usr/bin/env bash
+usage() { echo "$0 usage:" && grep " .)\ #" $0; exit 0; }
```

Comment:
Cool! I've not seen this before.

Could you please also add the usage to a comment at the top of this file, so it's easy to see how to run this?

Reply:
It actually pulls the comments from the cases below to build the usage and it looks like:
```
$ ./service/local-dev/run_server.sh
./service/local-dev/run_server.sh usage:                                                                                                                         
    a) # Disable authentication.                                                                                                                                 
    v) # Use Verily underlays.                                                                                                                                   
    h | *) # Display help.
```
Is it sufficient to have the flags documented that way? It's nice not to have duplicate documentation so if I was going to separately add a usage block at the top I'd want that to double as comments and output instead of also having this.

Reply:
I don't think you need to duplicate everything, but it wasn't immediately clear to me how to specify an option. I kept trying `./service/local-dev/run_server.sh a` (i.e. without the required dash). And the usage doesn't print out if you just call the script `./service/local-dev/run_server.sh`, that uses the defaults instead.

So maybe another approach is to change the script so `./service/local-dev/run_server.sh` prints out the usage, and somewhere note that these are flags, not arguments?

Reply:
How about this?